app/src/main/java/com/example/desafio/adapter/MyPageAdapter.kt

```diff
@@ -28,9 +29,14 @@ class MyPageAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
             }
```

Comment:
seguindo a sugestão da lista abaixo aqui também ficaria assim : 

```suggestion
  return FragmentOne(categoriesList[position])
```